pkg/alpine/mirrors/3.9

```diff
@@ -26,6 +26,7 @@ elfutils-libelf
 findutils
 flex
 gcc
+gcc-doc
```

Comment:
This doesn't seem to be used anymore -- please drop it

Reply:
removed